requirements.txt

```diff
@@ -1 +1,2 @@
 python_prctl
+psutil
```

Comment:
This done according to Travis-CI doc and common conventions. PyBitmessage itself has no dependencies.

Reply:
So the docs [say](https://docs.travis-ci.com/user/languages/python/#Dependency-Management):
```
You can manually override this default install phase, for example:
install: pip install --user -r requirements.txt
```
So we could have something like `pip install .[testing]` it seems. I'm all in favour of pulling in requirements from txt files to keep it DRY (instead of the other way round as I did with checkdeps.py) I'd just ask that we call it test_requirements.txt or requirements-dev.txt or something since, as you say, PyBitmessage itself has no dependencies so requirements.txt should be empty or missing.